Formula/open-coarrays.rb

```diff
@@ -0,0 +1,57 @@
+class OpenCoarrays < Formula
+  desc "open-source coarray Fortran ABI, API, and compiler wrapper"
+  homepage "http://opencoarrays.org"
+  url "https://github.com/sourceryinstitute/opencoarrays/releases/download/1.8.2/OpenCoarrays-1.8.2.tar.gz"
+  sha256 "dc5b98a62de85e38500f431b05290415d4a4a87001c0ba03fbe57403ad62010b"
+
+  head "https://github.com/sourceryinstitute/opencoarrays.git"
+
+  option "without-test", "Skip build time tests (not recommended)"
```

Comment:
Well.... I don't know... Sometimes users would rather have the software because it's hard to get one's hands on Fortran compilers supporting recent 2008 and 2015 (or 16? or 17? not sure what the standard ended up getting called...) coarray fortran features. On some systems, e.g., when linux brew pulls this in down stream, you might encounter test failures... but after seeing that you may still want to install OpenCoarrays and proceed cautiously with the library anyway... If you don't provide this build time option, than a test failure will completely prevent users from installing via Homebrew (or Linuxbrew). For this reason I would much rather keep this option in.